src/plugins/BeforeUnloadPlugin.js

```diff
@@ -36,4 +37,8 @@ export class BeforeUnloadPlugin extends BaPlugin {
 			observe(store, (state) => state.tools.current, onToolChanged, false);
 		}
 	}
+
+	_getTools() {
+		return [Tools.DRAW, Tools.MEASURE, Tools.MEASURE];
```

Comment:
could be reduced to `[Tools.DRAW, Tools.MEASURE]`

Reply:
Thx must be `ROUTING` of course